src/common/commonutils/FileUtils.c

```diff
@@ -893,6 +916,11 @@ int RenameFile(const char* original, const char* target, void* log)
         status = (0 == errno) ? ENOENT : errno;
     }
 
+    if (DetectSelinux(log))

```

Comment:
>(DetectSelinux(log)) [](http://example.com/codeflow?start=7&length=20)

Should we also limit RestoreSelinuxContext() for just the affected system files? 

Also, should we condition this on the successful rename (basically to not do RestoreSelinuxContext when the target is not modified)?